src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificatesTabCoordinator.swift

```diff
@@ -413,4 +354,40 @@ final class HealthCertificatesTabCoordinator {
 			self.viewController.present(alert, animated: true, completion: nil)
 		}
 	}
+	
+	private func showAdmissionScenarios() {
+		let result = self.cclScenariosHelper.showAdmissionScenarios()
```

Comment:
```suggestion
		let result = cclScenariosHelper.showAdmissionScenarios()
```

No explicit self needed